### DISCUSSION TITLE
[MacOS] Motions with ' or " eat up the character under the cursor when using dead keys

### DISCUSSION THREAD

**Discussion by lolsokje** [JetBrains/ideavim#821] [Q&A]
I use an input source with dead keys as I often have to type characters like `è`, `é`, etc. However, doing so with IdeaVim motions causes the character under the cursor to be removed when using motions like `vi"`/`vi'`.

Given the following piece of text (with my cursor on the `s`)
```
"this is a test"
             ^
```
The `s` will get removed when using `vi"` in IdeaVim. In regular Neovim however, the entire text is selected as it should, without removing any characters.

I honestly don't know if this is a JetBrains/PhpStorm issue or an IdeaVim issue, but at the moment it's impossible for me to use these motions.

Is there any way to fix this behaviour without changing input source?

**Comment by lippfi** [comment]
There is a YouTrack ticket for this issue - https://youtrack.jetbrains.com/issue/VIM-1503/Selects-and-motions-with-or-eats-up-characters-Mac-only-U.S.-International-keyboard-layout
